things/counter-thing/README.md

```diff
@@ -0,0 +1,28 @@
+# Counter Thing
+
+A simple counter implementation for the Eclipse Thingweb Test Things project.
+
+## Description
+
+This counter thing provides increment, decrement, and reset operations, exposing its state and changes through HTTP.
+
+## Features
+
+-   Increment, decrement, and reset operations
+-   Read-only count property
+-   SVG and PNG image property representations. These are provided for testing different content types
+-   Event emission on count change
+-   TypeScript implementation using node-wot
+
+## Protocol Support
+
+-   HTTP
+
+## Programming Language and Framework
+
+-   **Language**: TypeScript
+-   **Framework**: node-wot
+
+## Usage
+
+The counter exposes properties and actions for interacting with the count value. See the [Thing Description](https://github.com/eclipse-thingweb/test-things/blob/main/things/counter-thing/counter-thing.td.json) for details on endpoints and data formats. 
```

Comment:
Counter has been added in https://github.com/eclipse-thingweb/test-things/pull/73

We should add a proper link to it once fully hosted